cypress/features/regression/accessibility/a11y-second-part.test.js

```diff
@@ -0,0 +1,3 @@
+import tests from "../../../support/step-definitions/accessibility/a11y-utils";
+
+tests(301, 600);
```

Comment:
[Slice does not include start](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) so it should start at `300`
```suggestion
tests(300, 600);
```